src/types/api/interaction.ts

```diff
@@ -31,8 +36,24 @@ export interface DiscordInteractionData {
   id: string;
   /** the name of the invoked command */
   name: string;
+  /** converted users + roles + channels */
+  resolved?: DiscordInteractionDataResolved;
   /** the params + values from the user */
-  options: DiscordInteractionDataOption[];
+  options?: DiscordInteractionDataOption[];
+}
+
+export interface DiscordInteractionDataResolved {
```

Comment:
```suggestion
export interface DiscordApplicationCommandInteractionDataResolved {
```
To be consistent with documentation

Reply:
So 2 options:
1. Keep it as it is now
2. Name it `DiscordApplicationCommandInteractionDataResolved` but also rename all other interaction interfaces

Reply:
I would say option 3 is best, change this one real quick and then change others in another pr in future in v11